docs/advanced/secure-setup.md

```diff
@@ -0,0 +1,314 @@
+# <img class="dcr-icon" src="/img/dcr-icons/LockEye.svg" /> Secure Setup
+
+Last updated for CLI release v{{ cliversion }}.
+
+---
+
+This document will guide how to setup a two system method for holding your Decred securely.
+Some of the commands given will be specific to Debian based systems, but the principles/tools can be applied to almost any OS.
+
+**Please note that this setup requires two computers.**
+
+Although focused on Decred and Raspberry Pi, this guide can also be used for
+more general hardware devices and almost any cryptocurrency.
+
+---
+
+## Scope and Limitations
+
+The scope of this guide is to secure your cryptocurrency holdings from common malware and light threats.
+If you are a normal cryptocurrency holder and follow this guide exactly, it should give a good level of security.
+
+The guide assumes some general technical knowledge and that you can fix any hitches you come across.
+
+**This setup will not protect you against a state-level attacker or a persistent threat.**
+
+[General Security](general-security.md) offers more techniques to keep your system secure.
+
+---
+
+## The Setup
+
+This will be a multi-system setup.
+
+![Network diagram](/img/secure-setup.png)
+
+* A laptop or desktop that will be running the dcrd node, referred to as `Computer-A`.
```

Comment:
I think it would be better to immediately list the `Benefits of this Setup` section right after the picture. Readers who are "just checking it out" will probably be wondering "Why would I want to do this?" Also, the bullet points here don't necessarily explain the image above, so they feel disjointed.

My recommendation would be to delete these three bullet points, move directly into the `Benefits of this Setup` section below, and then go into further details about the hardware next.